.github/workflows/build_jammy_humble_essentials.yml

```diff
@@ -0,0 +1,74 @@
+name: Docker - Build Jammy/Humble Essentials
+
+on:
+  pull_request:
+    branches: [ "main" ]
```

Comment:
Probably get rid of the `branches: [ "main" ]` (as this currently implies anything that merges into main will encounter this workflow)

Reply:
So my concern is [this](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions): 

> If you specify multiple activity types, only one of those event activity types needs to occur to trigger your workflow.

My interpretation of the relationship between your two filters, "branches" / "paths", is OR, not AND. If someone PRs a change to `main` but didn't touch the Dockerfile, then currently this workflow would trigger (and it shouldn't)

Reply:
That makes sense! `branches: [ "main" ]` has been removed.